cpp/oneapi/dal/backend/primitives/test/ndarray.cpp

```diff
@@ -489,6 +506,47 @@ TEST_M(ndarray_test, "can assign ndarray", "[ndarray]") {
     sycl::free(x_ptr, queue);
 }
 
+TEST_M(ndarray_test, "can be flattened with host usm", "[ndarray]") {
+    DECLARE_TEST_POLICY(policy);
+    auto& queue = policy.get_queue();
+
+    auto [x, event] = ndarray<float, 2>::ones(queue, { 7, 5 });
```

Comment:
The test name contradicts with the test itself. By default, `ndarray` uses shared USM for allocation.

Reply:
Fixed